project/internal/pdb2/genqlient.graphql

```diff
@@ -93,11 +93,10 @@ query getLabs {
 	}
 }
 
-query getBookingEvents($start: DateTime, $end: DateTime, $resources: [ID!]) {
+query getBookingEvents($start: DateTime!, $end: DateTime!, $resources: [ID!]) {
 	bookingEvents(filterBy: {
 		resource: { lab: { id: {in: $resources} } },
-		start: { after: $start },
-		end: { before: $end }
+		timeRange: { overlaps: { start: $start, end: $end} }
```

Comment:
Could also just use `start: { after: $start, before: $end }` if you don't want the overlapping events that start before `$start`.